amandas-project/src/components/Cell.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const CELL_SIZE = 20
```

Comment:
It is a better practice to define all of this variables on a separate file or in the main file to be able to easily change them if required